imap/http_h2.c

```diff
@@ -874,6 +874,11 @@ HIDDEN int http2_start_session(struct transaction_t *txn,
         txn->strm_ctx = strm;
         txn->flags.ver = VER_2;
         ptrarray_add(&txn->done_callbacks, &stream_free);
+
+        /* Tell syslog our stream-id */
+        buf_printf(&txn->buf, "%d", strm->id);
```

Comment:
The rest of the code above uses buf instead of txn->buf, does that matter?

Reply:
It doesn't, but I should stay consistent